extensions/machine-learning-services/package.json

```diff
@@ -27,28 +28,66 @@
   "contributes": {
     "commands": [
       {
-        "command": "ml.command.managePackages",
-        "title": "%ml.command.managePackages%"
+        "command": "mls.command.managePackages",
+        "title": "%mls.command.managePackages%"
+      },
+      {
+        "command": "mls.command.odbcdriver",
+        "title": "%mls.command.odbcdriver%"
+      },
+      {
+        "command": "mls.command.mlsdocs",
+        "title": "%mls.command.mlsdocs%"
+      }
+    ],
+    "dashboard.tabs": [
+      {
+        "id": "ml-server",
+        "description": "Machine Learning Services",
+        "provider": "MSSQL",
+        "title": "Machine Learning Services",
+        "when": "connectionProvider == 'MSSQL' && !mssql:iscloud",
+        "container": {
+          "grid-container": [
+            {
+              "name": "%title.configurations%",
+              "row": 0,
+              "col": 0,
+              "widget": {
+                "modelview": {
+                  "id": "ml.tasks"
+                }
+              }
+            },
+            {
+              "name": "%title.tasks%",
+              "row": 0,
+              "col": 1,
+              "widget": {
+                "tasks-widget": [
+                  "mls.command.managePackages",
+                  "mls.command.odbcdriver",
+                  "mls.command.mlsdocs"
+                ]
+              }
+            }
+          ]
+        }
       }
     ]
   },
   "dependencies": {
-    "vscode-nls": "^4.0.0"
+    "vscode-nls": "^4.0.0",
+    "vscode-languageclient": "^5.3.0-next.1"
```

Comment:
Why are you using this specific one?

Reply:
copied from others. and it's not specific since it has ^ . it will pick up any version upper than this

---
In reply to: [362899094](https://github.com/microsoft/azuredatastudio/pull/8796#discussion_r362899094) [](ancestors = 362899094)